### PR TITLE
Readme Patch

### PR DESCRIPTION
**Pull request by newklus**
1. add some item.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,53 @@
-Absolute
-=============
-
+# Absolute
 [![Linux/Max Build Status](https://travis-ci.org/lunchclass/absolute.svg?branch=master)](https://travis-ci.org/lunchclass/absolute/branches)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/099u4iekeny4lpsa/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
+[![Windows Build status](https://ci.appveyor.com/api/projects/status/pmac6fwbclavu8n0/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
```
Comment (romandev) [inline README.md]: Why do we change this link?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,53 @@
...
+[Contributing to Absolute guide](http://lunchclass.io).
 
 ## Getting Started
 
+### System Requirements
+#### Linux
+
+For Ubuntu, you'll need the following development dependencies:
+
+    sudo apt-get install g++ git make python wget
```
Comment (romandev) [inline README.md]: We only need git for absolute.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,53 @@
...
 ## Getting Started
 
+### System Requirements
+#### Linux
+
+For Ubuntu, you'll need the following development dependencies:
+
+    sudo apt-get install g++ git make python wget 
+
+Build instructions for Ubuntu 14.04 LTS can be found [here](docs/Ubuntu14.md)
```
Comment (romandev) [inline README.md]: We don't need this sentense I think..?

File: README.md [hunk 4]
```diff
@@ -1,19 +1,53 @@
...
+TODO Input Flow Image
+
+### Features
+* client(web): 
+* push server: 
+* payment: 
+
+## Contributing to Absolute
+Contributions to Absolute are welcomed and encouraged! Please see the
+[Contributing to Absolute guide](http://lunchclass.io).
```
Comment (romandev) [inline README.md]: We need another file e.g. CONTRIBUTION.md

File: README.md (hunk 1 above)
Comment (newklus) [reply README.md]: I didn't know that the build state address of my repo was different from the build state address of origin. I'll fix it soon

File: README.md (hunk 4 above)
Comment (newklus) [reply README.md]: I will delete it and make it contribution.md later.

File: README.md (hunk 2 above)
Comment (newklus) [reply README.md]: ok. I'ii fix it.

File: README.md (hunk 3 above)
Comment (newklus) [reply README.md]: ok. I'ii remove it

**Pull request closed without merging**